### PR TITLE
FHDDS-879: removed amendments ref, changed subject line and heading a…

### PR DESCRIPTION
**Pull request by nubz**
…nd tidied html

`[info] ScalaTest
[info] Run completed in 29 seconds, 895 milliseconds.
[info] Total number of tests run: 134
[info] Suites: completed 2, aborted 0
[info] Tests: succeeded 134, failed 0, canceled 0, ignored 0, pending 0
[info] All tests passed.
[info] Passed: Total 134, Failed 0, Errors 0, Passed 134
[success] Total time: 36 s, completed 07-Mar-2018 12:04:52
`

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
+<p style="margin: 0 0 30px; font-size: 19px;">See the guidance on GOV.UK to find out:</p>
+<ul style="margin: 0 0 30px; font-size: 19px;">
     <li>what you need to do while HMRC processes your application</li>
-    <li>how to make changes to your application</li>
     <li>how to contact HMRC about your application</li>
 </ul>
-</p>
-
 <p style="margin: 0 0 30px; font-size: 19px;">From the HMRC Fulfilment House Due Diligence Scheme</p>
+<p style="margin: 0 0 30px; font-size: 19px;">Do not reply to this email.</p>
```
Comment (tev1622) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: not needed as its already in the footer

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt [hunk 2]
```diff
@@ -15,9 +15,12 @@ Help using this service
...
 
 - what you need to do while HMRC processes your application
-- how to make changes to your application
 - how to contact HMRC about your application
 
 From the HMRC Fulfilment House Due Diligence Scheme
 
+--
+
+Do not reply to this email.
```
Comment (tev1622) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt]: ditto

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html (hunk 1 above)
Comment (nubz) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: ah thanks - I saw that but noticed the don't reply instruction was only if user's were not sure if email was really from HMRC or not. We can take it out.

**Comment by nubz** [comment]
As per the request of our content designer (https://jira.tools.tax.service.gov.uk/browse/DC-1368) we have re-instated the "Do not reply to this email." in the body of the email

**Pull request merged**